styles/style.css

```diff
@@ -95,7 +97,25 @@ p {
 
 /* Buttons */
 
-  /* INSERT BUTTON STYLES HERE */
+  button{
+    background-color: var(--orange-dark);
+    color:var(--orange-light);
+    padding:5px;
+    border-radius:5px;
+    margin:-20px 20px 0 -20px;
+  }
+  .button_vol{
```

Comment:
Well-done on not repeating the same styles!

I would suggest a more generic name like `button__primary` and `button__secondary` etc. The reason is that you may want to use this button for other purposes with the same style.